testing/dart/window_hooks_integration_test.dart

```diff
@@ -115,7 +121,24 @@ void main() {
       });
 
       window.onMetricsChanged();
-      _updateWindowMetrics(0.1234, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0);
+      _updateWindowMetrics(
+        0.1234, // DPR
+        0.0,    // width
```

Comment:
From a verification perspective, I'm wondering if it's a good idea to use so many `0.0` values. Once could imagine that a couple of these values got mixed up in the implementation, but because they're all `0.0`, such a test would not catch any of those mistakes. You might consider using different values for every entry.

Reply:
This particular test isn't exercising these values, it's just exercizing the zone it's in.  There's another test that uses different values below.